feature/account/edit/src/main/kotlin/app/k9mail/feature/account/edit/ui/server/settings/EditIncomingServerSettingsNavHost.kt

```diff
@@ -49,6 +51,7 @@ fun EditIncomingServerSettingsNavHost(
         }
         composable(route = NESTED_NAVIGATION_ROUTE_VALIDATE) {
             ServerValidationScreen(
+                title = stringResource(id = R.string.account_server_settings_incoming_top_bar_title),
```

Comment:
This could be a flag `showToolbar` and the title assignment could be moved to the `SeverValidationScreen`

Reply:
This would require `:feature:account:server:validation` to have a dependency on `:feature:account:server:settings`, just so it could access the string. It would also require an additional parameter to distinguish between checking incoming and outgoing server settings.

How about different names for the screens (`SetupServerValidationScreen` vs. `SettingsServerValidationScreen`) instead of the magic optional `title` parameter in this PR?
https://github.com/thunderbird/thunderbird-android/compare/server_validation_with_toolbar_2